src/core_tests/queries.rs

```diff
@@ -74,12 +73,18 @@ async fn legacy_query(#[case] include_history: bool) {
         .await
         .unwrap();
     };
+    let clear_eviction = || async {
+        let t = core.poll_workflow_task(TEST_Q).await.unwrap();
+        core.complete_workflow_task(WfActivationCompletion::empty(t.run_id))
+            .await
+            .unwrap();
+        first_wft().await;
```

Comment:
I would leave `first_wft` out of this function, it doesn't have anything to do with clearing the eviction.